libraries/Microsoft.Bot.Builder/Microsoft.Bot.Builder.csproj

```diff
@@ -21,6 +21,10 @@
 		<Description>Library for building bots using Microsoft Bot Framework Connector</Description>
 		<Summary>Library for building bots using Microsoft Bot Framework Connector</Summary>
 	</PropertyGroup>
+
+	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Documentation|AnyCPU'">
+	  <DocumentationFile></DocumentationFile>
+	</PropertyGroup>
```

Comment:
I thought you just removed these?

Reply:
I did in another branch, it will all be good in master.